pkg/registry/generic/registry/store_test.go

```diff
@@ -1091,17 +1130,19 @@ func TestStoreWatch(t *testing.T) {
 			}
 			wi.Stop()
 		}
-
+		destroyFunc()
 		server.Terminate(t)
 	}
 }
 
-func newTestGenericStoreRegistry(t *testing.T, hasCacheEnabled bool) (*etcdtesting.EtcdTestServer, *Store) {
+func newTestGenericStoreRegistry(t *testing.T, hasCacheEnabled bool) (*etcdtesting.EtcdTestServer, func(), *Store) {
 	podPrefix := "/pods"
 	server := etcdtesting.NewEtcdTestClientServer(t)
 	strategy := &testRESTStrategy{api.Scheme, api.SimpleNameGenerator, true, false, true}
+
 	codec := testapi.Default.StorageCodec()
 	s := etcdstorage.NewEtcdStorage(server.Client, codec, etcdtest.PathPrefix(), false, etcdtest.DeserializationCacheSize)
+	destroyFunc := func() {}
```

Comment:
Yeah. Dismissed.
It creates http transport and haven't recycled it here:
https://github.com/kubernetes/kubernetes/blob/ef0c9f0c5b8efbba948a0be2c98d9d2e32e0b68c/pkg/storage/etcd/testing/utils.go#L238